apps/web/src/components/utils/hooks/useEthersWalletClient.tsx

```diff
@@ -8,7 +9,7 @@ const useEthersWalletClient = (): {
   };
   isLoading: boolean;
 } => {
-  const { data, isLoading } = useWalletClient();
+  const { data, isLoading } = useWalletClient({ chainId: CHAIN_ID });
```

Comment:
So `useWalletClient` not picking the chain id automatically?

Reply:
I'm not really sure why. I just know that after making these changes, I was able to load my inbox and stopped receiving signature errors.